.env.example

```diff
@@ -2,4 +2,4 @@ PORT=3000
 NODE_ENV=development
 TOKEN_SECRET=changeMe
 DATABASE_URL=postgres://dev:dev@localhost/FLASH?sslmode=disable
-NEXT_PUBLIC_GOOGLE_MAPS_API_KEY = "AIzaSyC3q7FbfvJEgmnvm2Lq2dMKurjfBFqWQb0"
+GOOGLE_MAPS_API_KEY = "AIzaSyC3q7FbfvJEgmnvm2Lq2dMKurjfBFqWQb0"
```

Comment:
Your .env.example should not have a real value in it.

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#envexample-and-env